### PR TITLE
Real Analysis notes. For April 18, 2017.

### PR DESCRIPTION
**Pull request by MareoRaft**
Please oh please accept me!

### CODE REVIEW

File: Real Analysis/real-analysis-notes.tex [hunk 1]
```diff
@@ -3516,6 +3522,155 @@ \subsection{Density of Continuous Functions}
...
 \subsection*{(4/18/17) Lecture 23}
+\subsection{Product Measures and Fubini}
+\begin{thm}[Practical Carathéodory Theorem]
+  If $X$ is a set, $\E$ is an elementary family of subsets of $X$, and $\mu \from \E \to [0, \oo]$ is a function such that
+  \begin{itemize}
+    \item $\mu(\emptyset) = \emptyset$
+    \item $\mu$ is \s additive on $\E$ (but the disjoint union need not land back in $\E$)
+    \item $\mu$ is \s finite,
+  \end{itemize}
+  then there exists a unique measure $\mu$ on $\M(\E)$ that extends the above $\mu$.  Moreover, for any $A \in \M(\E)$, $$\mu(A) = \inf\sum_{n=1}^N \mu(E_n)$$ where each $\{E_n\}_{n=1}^N$ is a finite disjoint cover of $A$.
```
Comment (ghseeli) [inline Real Analysis/real-analysis-notes.tex]: Perhaps this is silly, but what exactly is the inf over? All finite disjoint covers of A?

File: Real Analysis/real-analysis-notes.tex [hunk 2]
```diff
@@ -3516,6 +3522,155 @@ \subsection{Density of Continuous Functions}
...
+\end{thm}
+\begin{proof}
+  HW 8.
+\end{proof}
+\begin{defn}
+  If $\M$ and $\cN$ are \s algebras, then the \de{product \s algebra} is $$\M \ox \cN := \M(\{A \x Y \st A \in \M\}\union\{X \x B \st B \in \N\}),$$ that is, the \s algebra generated by that set.  This is also equal to the \s algebra generated by $\{A \x B \st A \in \M, B \in \cN\}$.
+\end{defn}
```
Comment (ghseeli) [inline Real Analysis/real-analysis-notes.tex]: This definition is already in the text; see lines 143-157.

File: Real Analysis/real-analysis-notes.tex [hunk 3]
```diff
@@ -3516,6 +3522,155 @@ \subsection{Density of Continuous Functions}
...
+        \sum \left(\chi_{A_n}(x)\int_Y\chi_{B_n}(x)\d\nu(y)\right) &= \chi_A(x)\int_Y \chi_B(y) \d\nu(y) \\
+        \sum \chi_{A_n}(x)\nu(B_n) &= \chi_A(x)\nu(B) \\
+        \int_X \left(\sum \chi_{A_n}(x)\nu(B_n)\right) \d\mu(x) &= \int_X \chi_A(x)\nu(B)\d\mu(x) \\
+        \sum \left(\int_X \chi_{A_n}(x)\nu(B_n) \d\mu(x)\right) &= \int_X \chi_A(x)\nu(B)\d\mu(x) \\
+        \sum \left(\int_X \chi_{A_n}(x) \d\mu(x)\right)\nu(B_n) &= \left(\int_X \chi_A(x)\d\mu(x)\right)\nu(B) \\
+        \sum \mu(A_n)\nu(B_n) &= \mu(A)\nu(B)
+      \end{align*}
+    \end{itemize}
+  \end{enumerate}
+  Finally, I have a big feeling that we have satisfied the hypothesis of the Practical Catheodary Theorem, and we simply invoke that theorem to get $(\mu\ox\nu)$ for the entire $\M\ox\cN$, not just $\E$.
```
Comment (ghseeli) [inline Real Analysis/real-analysis-notes.tex]: Is this just a feeling?

File: Real Analysis/real-analysis-notes.tex [hunk 4]
```diff
@@ -3516,6 +3522,155 @@ \subsection{Density of Continuous Functions}
...
+  \end{enumerate}
+\end{prop}
+\begin{proof}
+  \mbox{}
+  \begin{enumerate}[label=(\arabic{*})]
+    \item Let $f$ be as described.
+    $$\begin{tikzcd}
+      &Y \arrow[rr, bend left, "f_x"] \arrow[r, "\iota_x"'] &X \x Y \arrow[r, "f"'] &S \\
+      &y \arrow[r, mapsto] &(x,y) \arrow[r, mapsto] &f(x,y)
+    \end{tikzcd}$$
```
Comment (ghseeli) [inline Real Analysis/real-analysis-notes.tex:3627]: Yay pictures!

**Comment by ghseeli** [comment]
@MareoRaft Thank you for writing this up! It is incredibly helpful. If my comments are not addressed in my handwritten notes, then I can just work on cleaning up the things I commented on. Otherwise, I think we can go ahead and merge this and cover it in a separate pull request.

File: Real Analysis/real-analysis-notes.tex (hunk 1 above)
Comment (MareoRaft) [reply Real Analysis/real-analysis-notes.tex]: Yes.  This is addressed in my next commit (coming soon).

File: Real Analysis/real-analysis-notes.tex (hunk 2 above)
Comment (MareoRaft) [reply Real Analysis/real-analysis-notes.tex]: Thank you for the catch.

**Pull request merged**